File: maple/maple-user-web/src/main/webapp/js/service/workDataService.js
//服务层
app.service('workDataService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../workdata/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../workdata/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../workdata/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../workdata/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../workdata/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../workdata/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../workdata/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: maple/maple-type-interface/src/main/java/com/maple/service/WorkDataService.java
package com.maple.service;

import com.maple.pojo.Workdata;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface WorkDataService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Workdata> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Workdata Workdata);
	
	
	/**
	 * 修改
	 */
	public void update(Workdata Workdata);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Workdata findOne(int id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(int[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Workdata Workdata, int pageNum, int pageSize,String name);
	
}
